archeryutils/targets.py

```diff
@@ -18,8 +21,32 @@
     "Beiter_hit_miss",
     "Worcester",
     "Worcester_2_ring",
+    "Custom",
 ]
 
+# TypeAlias (annotate explicitly in py3.10+)
+FaceSpec = Mapping[float, int]
+
+_rnd6 = partial(round, ndigits=6)
+
+
+class Quantity(NamedTuple):
+    """
+    Dataclass for a quantity with units.
+
+    Can be used in place of a plain tuple of (value, units)
+
+    Attributes
+    ----------
+    value: float
+        Scalar value of quantity
+    units: str
+        Units of quantity
+    """
```

Comment:
Done on extra examples. Do you mean type hinting the target __init__ as taking `Quantity` instead of `tuple[float, str]` for its parameters? If so I don't think thats what we want to do here. On the maxim of "Accept the broadest possible types for parematers, return the most specific type", changing the signature to take Quantity will throw type errors everywhere in mypy when a plain tuple is used, and the plain tuple works absolutely fine (in fact we are parsing it and returning in a standardised form anyway). 

We could add a type alias (`QuantityLike`) for the repetitive use of `Union[float, tuple[float, str]]`, I think its on the edge of that being worthwhile to do at the moment.

Reply:
Ah, sorry, I meant to the top level `__init__.py` so that it is available to users when they `import archeryutils as au` as something like `au.Quantity`.